packages/hint/docs/user-guide/concepts/configurations.md

```diff
@@ -1,58 +1,75 @@
+A configuration is a way to share .hintrc values for different use
+cases, such as related hints, URLs to ignore, shared organization
+standards, etc. When installing a configuration, all of its dependencies
+(hints, connectors, formatters, parsers) should be installed
+automatically as well.
+
+Conveniently, any configuration you choose when running
+`npm create hintrc` is automatically installed _and_ added to your
+.hintrc for you.
+
+To add a configuration other than (or in addition to) those offered by
+running `npm create hintrc`, first install its package. Make sure the
+package name begins with `@hint/configuration-`,
+`webhint-configuration-`, or `@namespace/webhint-configuration-`. Once
+installed, update your .hintrc to use it by adding the package name to
+the `extends` array. Given a package called
+`webhint-configuration-example1`, add the following:
 
 ```json
 {
-    "extends": ["example1"]
+    "extends": ["webhint-configuration-example1"]
 }
 ```
 
-3. You are done!
-
-The property `extends` is `string[]` so you can extend from
-multiple configuration packages:
+Because the property `extends` is an array of strings, you can extend
+from multiple configuration packages. For example, if you wish to add
+`@hint/configuration-web-recommended`, `webhint-configuration-example2`,
+and `@orgName/webhint-configuration-example3`, your `extends` value will
+look like this:
 
 ```json
 {
-    "extends": ["example1", "example2"]
+    "extends": [
+        "@hint/configuration-web-recommended",
+        "webhint-configuration-example2",
+        "@orgName/webhint-configuration-example3"
+    ]
 }
 ```
 
-The priority applies from left to right. Any values in your `.hintrc`
-file will take precedence. For example, the following will always use
-the `summary` formatter:
+Configuration priority applies from left to right. Any values in your
+own .hintrc file will take precedence over those in an included
+configuration package. For example, the following will always use the
+`summary` formatter, regardless of the content of
+`webhint-configuration-example1` and `webhint-configuration-example2`
+configurations: (see Notes for more details)
 
 ```json
 {
-    "extends": ["example1", "example2"],
+    "extends": [
+        "webhint-configuration-example1",
+        "webhint-configuration-example2"
+    ],
     "formatters": ["summary"]
 }
 ```
 
 Notes:
 
-* If you define the property `formatters` when extending
-  a configuration, the formatters in the configuration will be
-  _replaced_ with the value you have defined.
+* If you define the property `formatters` when extending a
+  configuration, the formatters in the configuration will be _replaced_
+  with the value you have defined.
+
+* If you define the property `parsers` when extending a configuration,
+  the parsers in the configuration will be _appended_ to the values you
+  have defined.
+
+If you want to implement your own custom configuration, visit the
+[contributor guide][].
```

Comment:
Is `[contributor guide][]` the same as `[contributor guide][contributor guide]`?

Reply:
yeah! isn't that wild? i've been a markdown devotee forever and idk if it's just GFM that handles links like that or most/all markdown parsers but i just learned about this simplified (and more readable) style last month.